---
title: SARS-CoV-2
permalink: /primers/sars-cov-2/
wishlist: yes
---

# SARS-CoV-2

[SARS-CoV-2](https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2) is a highly contagious [coronavirus](https://en.wikipedia.org/wiki/Coronavirus), causing the [COVID-19](https://en.wikipedia.org/wiki/Coronavirus_disease_2019) disease. The common symptoms include fever, cough, and shortness of breath. It was first discovered in the Chinese city of Wuhan, and has since spread across the globe. The [World Health Organization](https://who.int) declared the 2019-2020 outbreak as a pandemic, and many regions have imposed drastic measures to contain the spread of the virus.

## Key points

  * The [genome](https://www.nature.com/articles/s41586-020-2008-3) of the virus has been published in early February 2020.

  * The [spike (S) proteins](https://www.sinobiological.com/research/virus/hcov-spike-protein-overview) gather on the surface of the virion and form the crown-like shape, leading to the *corona* name.

  * The [ACE2](https://en.wikipedia.org/wiki/Angiotensin-converting_enzyme_2) enzyme serves as the entrypoint for some coronavirus spike proteins into human cells in the lungs, arteries, heart, kidney, and intestines.

  * Proteins exist as unfolded random coils when translated from a sequence of mRNA to a linear chain of amino acids. These coils lack any stable three-dimensional structure. As the coils are being synthesized by ribosomes, they begin to fold into their native structures. This is crucial to understand potential docking points for drugs into the virus. Computational studies of this [protein folding](https://en.wikipedia.org/wiki/Protein_folding) include the prediction of protein stability, kinetics, and structure.

  * [Folding@Home](https://en.wikipedia.org/wiki/Folding@home) and [Rosetta@Home](https://en.wikipedia.org/wiki/Rosetta@home) are two computational protein folding projects, which harness the distributed computing power of volunteer resources.
